pkg/annotation/aws.go

```diff
@@ -0,0 +1,14 @@
+package annotation
+
+// AWSUpdateMaxBatchSize is the aws update annotation used for configuring
+// maximum batch size for instances during ASG update.
+// The value can be either a whole number specifying the number of instances
+// or a percentage of total instances as decimal number ie: `0.30` for 30%.
+// https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-attribute-updatepolicy.html#cfn-attributes-updatepolicy-rollingupdate-maxbatchsize
+const AWSUpdateMaxBatchSize = "aws.giantswarm.io/update-max-batch-size"
+
+// AWSUpdatePauseTime is the aws update annotation used for configuring
+// time pause between rolling a single batch during ASG update.
+// The value must be in ISO_8601 format.
```

Comment:
Since the ISO 8601 standard is long and the duration format is not exactly intuitive, I would definitely give an example here that might work for some people.

```suggestion
// The value must be in ISO_8601 format, e. g. "PT5M" for five minutes.
```